src/shared/components/events-list/index.js

```diff
@@ -57,19 +57,22 @@ export default class EventsList extends Component {
               relevantEvents.map((event) => (
                   <li key={`event_${event.id}`} className={styles.eventItem}>
```

Comment:
One could argue it's serviceable tech debt, but I think this component could do with being split up into smaller components. The `render` is too big IMO